plugins/inputs/kinesis_consumer/kinesis_consumer.go

```diff
@@ -238,8 +260,47 @@ func (k *KinesisConsumer) Start(ac telegraf.Accumulator) error {
 	return nil
 }
 
+func gzipDecompress(data []byte) ([]byte, error) {
```

Comment:
Only cosmetics, but can you please change the name to `decompressGzip()` to match the other names starting with an action...